pkg/csi/manila/share.go

```diff
@@ -75,7 +90,7 @@ func getOrCreateShare(shareName string, createOpts *shares.CreateOpts, manilaCli
 		return share, 0, nil
 	}
 
-	return waitForShareStatus(share.ID, shareCreating, shareAvailable, false, manilaClient)
+	return waitForShareStatus(share.ID, []string{shareCreating, shareCreatingFromSnapshot}, shareAvailable, false, manilaClient)
```

Comment:
Awesome!

Reply:
Yeah, this looks right.  (@gouthampacha reminded me yesterday)

Reply:
Thank you both! I'll update this PR once I find out what do we do with updating gophercloud lib dependency.